js/algoliasearch/instantsearch.js

```diff
@@ -2,6 +2,10 @@ document.addEventListener("DOMContentLoaded", function (event) {
 	algoliaBundle.$(function ($) {
 
 		function makeUrlForInsights(baseUrl, objectID, queryID, indexName) {
+			if (algoliaConfig.ccAnalytics.enabled !== true) {
```

Comment:
can this be simplified to `algoliaConfig.ccAnalytics.enabled` ? unless it's not always a boolean